sdk/cognitivelanguage/azure-ai-language-conversations/swagger/README.md

```diff
@@ -78,14 +78,17 @@ directive:
   where: $.paths.*.*
   transform: |
     var operationId = $.operationId.replace(/_/g, "/").replace(/([a-z0-9])([A-Z])/g, "$1-$2").toLowerCase();
-    $.description = $.description + "\n\nSee https://learn.microsoft.com/rest/api/language/" + operationId + " for more information.";
+    var apiVersion = "2022-10-01-preview/"
```

Comment:
This could be automated. I can find a way if you don't want to. Worst case scenario, we pass the whole document in and enumerate the paths ourselves.